src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -72,15 +75,32 @@ export default class TranscriptArea {
     ulElement.class = 'mx-auto';
     transcriptContainer.appendChild(ulElement);
     transcriptLines.forEach((transcriptLine) => {
-      ulElement.appendChild(
-          TranscriptLineElement.createTranscriptLineElement(transcriptLine));
+      const transcriptLineElement =
+          TranscriptLineElement.createTranscriptLineElement(transcriptLine);
+      ulElement.appendChild(transcriptLineElement);
+      TranscriptArea.transcriptLineMap.set(
+          transcriptLine.transcriptKey.id, transcriptLineElement);
     });
   }
 
+  /** Increments the indicator corresponding to `transcriptLineKey` by 1. */
+  static incrementCommentIndicatorAt(transcriptLineKeyId) {
+    if (!TranscriptArea.transcriptLineMap.has(transcriptLineKeyId)) {
+      return;
+    }
+    const commentIndicatorElement =
+        TranscriptArea.transcriptLineMap.get(transcriptLineKeyId)
+            .commentIndicator;
+    commentIndicatorElement.innerText =
+        parseInt(commentIndicatorElement.innerText) + 1;
+    commentIndicatorElement.style.visibility = 'visible';
```

Comment:
nit: can this be a const?

Reply:
I think it's clearer what is happening to leave it as is if that's okay.